.github/workflows/test-security.yml

```
@@ -24,7 +23,9 @@ jobs:
           requirements-suffix: "-dev"
 
       - name: <Test> Check Python dependency security
-        run: safety check -r requirements.txt -r requirements-dev.txt -r requirements-build.txt -o bare
+        uses: pypa/gh-action-pip-audit@v1.1.0
```

まああまり凝った使い方してないですし、v1指定でも大丈夫かも！
```suggestion
        uses: pypa/gh-action-pip-audit@v1
```

---

https://github.com/pypa/gh-action-pip-audit/tags
多分v1タグが無いのでできない気がします。

---

あっ。。本当ですね。。
issue立てても良いかもだけど、ま、まあいいか。。